tests/new_upgrades/test_subscription.py

```diff
@@ -0,0 +1,205 @@
+        5. Create a content host.
+
+    :expectedresults:
+        1. Content host should be created.
+    """
+    target_sat = subscription_upgrade_shared_satellite
+    manifest = subscription_upgrade_manifest
+    with SharedResource(target_sat.hostname, upgrade_action, target_sat=target_sat) as sat_upgrade:
+        test_name = f'auto_attach_upgrade_{gen_alpha()}'
+        org = target_sat.api.Organization(name=f'{test_name}_org').create()
+        # Simultaneously uploading manifests to multiple orgs has a high failure rate, so stagger
+        # the uploads by ten seconds.
+        xdist_worker_splay = int(os.environ.get('PYTEST_XDIST_WORKER')[-1]) * 10
+        sleep(xdist_worker_splay)
+        target_sat.upload_manifest(org.id, manifest.content)
+        location = target_sat.api.Location(name=f'{test_name}_location').create()
+        library_id = int(
+            target_sat.cli.LifecycleEnvironment.list(
+                {'organization-id': org.id, 'library': 'true'}
+            )[0]['id']
+        )
+        lce = target_sat.api.LifecycleEnvironment(
+            name=f'{test_name}_lce', organization=org, prior=library_id
+        ).create()
+        if rhel_contenthost.os_version.major > 7:
+            rh_repo_id = target_sat.api_factory.enable_sync_redhat_repo(
+                constants.REPOS[f'rhel{rhel_contenthost.os_version.major}_bos'], org.id
+            )
+        else:
+            rh_repo_id = target_sat.api_factory.enable_sync_redhat_repo(
+                constants.REPOS[f'rhel{rhel_contenthost.os_version.major}'], org.id
+            )
+        rh_repo = target_sat.api.Repository(id=rh_repo_id).read()
+        assert rh_repo.content_counts['rpm'] >= 1
+        content_view = target_sat.publish_content_view(org, rh_repo, f'{test_name}_content_view')
+        content_view.version[-1].promote(data={'environment_ids': lce.id})
+        subscription = target_sat.api.Subscription(organization=org.id).search(
+            query={'search': f'name="{constants.DEFAULT_SUBSCRIPTION_NAME}"'}
+        )
+        assert len(subscription)
+        activation_key = target_sat.api.ActivationKey(
+            name=f'{test_name}_ak',
+            content_view=content_view,
+            organization=org.id,
+            environment=lce,
+            auto_attach=False,
+        ).create()
+        rhel_contenthost.api_register(
+            target_sat,
+            organization=org,
+            location=location,
+            activation_keys=[activation_key.name],
+        )
+        assert rhel_contenthost.subscribed
+        test_data = Box(
+            {
+                'rhel_client': rhel_contenthost,
+                'org': org,
+                'satellite': target_sat,
+            }
+        )
+        sat_upgrade.ready()
+        target_sat._session = None
+        yield test_data
+
+
+@pytest.mark.subscription_upgrades
+@pytest.mark.rhel_ver_list([7, 8, 9])
```

Comment:
`[7, 8, 9]` 
Surprised to see 7 on there as a parametrized option, do we still support RHEL 7 hosts? Also, should this parametrize to 10 as well?